src/v2/view-builder/ViewFactory.js

```diff
@@ -94,7 +100,19 @@ const VIEWS_MAPPING = {
     password: EnrollAuthenticatorPasswordView,
     'security_key': EnrollWebauthnView,
     phone: EnrollAuthenticatorPhoneView,
-    'security_question': EnrollAuthenticatorSecurityQuestion
+    'security_question': EnrollAuthenticatorSecurityQuestion,
+  },
+  'enroll-poll': {
```

Comment:
iirc, we were talking about using channel as another look up meta data
curious why we choose your current proposal?


```
  enroll-authenticator: {
     app: {
       email: ..,
       sms: ...
    }
  }
```

In practice, I think we could do 

```
// ViewFactory.js
  enroll-authenticator: {
     app: EnorllAuthenticatorOV,
  }

// EnorllAuthenticatorOV.js

initialize() {
  case channel of 
    'sms': this.renderEnrollOVSmS(); break;
    ....
}

```

Reply:
I think the form was named as enroll-poll and not 'enroll-authenticator' since the form itself doesn't enroll the authenticator.

Reply:
Checked offline with Haisheng and sounds like we are good with enroll-poll.